lib/Controllers/list.js

```diff
@@ -153,6 +155,10 @@ List.prototype.fetch = function(req, res, context) {
     model = model.scope(req.query.scope);
   }
 
+  if (transformOptions) {
+    transformOptions(options);
```

Comment:
What happened to `options = transformOptions(options)` ?

Reply:
While the methods itself returns the same reference of `option` I've decided to not use the assignment here as it give the wrong impression of that `option` object is not modified by the method, which is incorrect.